### PR TITLE
Add typo variations for 'serialize'

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8896,6 +8896,11 @@ descvriptions->descriptions
 desepears->disappears
 deserailize->deserialize
 deserializazion->deserialization
+deserialziation->deserialization
+deserialzie->deserialize
+deserialzied->deserialized
+deserialzies->deserializes
+deserialzing->deserializing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8903]: Can you also add the ise versions of all of this for the UK.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -25233,6 +25238,11 @@ sercives->services
 serciving->servicing
 sergent->sergeant
 serialiazation->serialization
+serialziation->serialization
+serialzie->serialize
+serialzied->serialized
+serialzies->serializes
+serialzing->serializing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25245]: Can you also add the ise versions of all of this for the UK.

**Review by peternewman — CHANGES_REQUESTED** [review]
Just the UK side to cover

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25233,6 +25238,11 @@ sercives->services
 serciving->servicing
 sergent->sergeant
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25239]:
```suggestion
sergent->sergeant
serailize->serialize
```

**Comment by peternewman** [comment]
I don't see any changes @luzpaz !

**Comment by luzpaz** [comment]
Apologies, I rebased this PR without looking at your feedback, @peternewman. Should be complete now. Please review

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -25278,6 +25290,7 @@ serach->search
 serached->searched
 serailisation->serialisation
 serailization->serialization
+serailize->serialize
 serailized->serialized
 serailze->serialize
 serailzed->serialized
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25296]: Would you mind adding the UK versions of these ones too please?

**Review by peternewman — CHANGES_REQUESTED** [review]
Just one outstanding comment

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:25484]: @luzpaz serailise and friends are still missing.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -25466,6 +25478,7 @@ serach->search
 serached->searched
 serailisation->serialisation
 serailization->serialization
+serailize->serialize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25481]: Like this, but also -ed etc.

```suggestion
serailise->serialise
serailization->serialization
serailize->serialize
```

**Review by peternewman — CHANGES_REQUESTED** [review]
Just clarifying the existing requests

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -25465,7 +25477,9 @@ ser->set
 serach->search
 serached->searched
 serailisation->serialisation
+serailise->serialise
 serailization->serialization
+serailize->serialize
 serailized->serialized
 serailze->serialize
 serailzed->serialized
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25485]: Can you add the the UK versions of these ones too please?

**Review by peternewman — CHANGES_REQUESTED** [review]
Just one outstanding group of UK equivalents please @luzpaz

**Review by bl-ue — APPROVED** [review]
LGTM — @peternewman PTAL

**Pull request merged**